source/shared/cpp/ObjectModel/ChoiceSetInput.cpp

```diff
@@ -6,6 +6,7 @@ using namespace AdaptiveCards;
 
 ChoiceSetInput::ChoiceSetInput() : BaseInputElement(CardElementType::ChoiceSetInput)
 {
+    PopulateKnownPropertiesSet();
```

Comment:
>PopulateKnownPropertiesSet [](start = 4, length = 26)

I would say have the base classes handle calling this so each subclass doesn't need to.

Reply:
We discussed this offline and we can't call virtual methods from the constructor.

---
In reply to: [171892893](https://github.com/Microsoft/AdaptiveCards/pull/1179#discussion_r171892893) [](ancestors = 171892893)